src/JustSaying/AwsTools/MessageHandling/Compression/GzipMessageBodyCompression.cs

```diff
@@ -3,9 +3,21 @@
 
 namespace JustSaying.AwsTools.MessageHandling.Compression;
 
-class GzipMessageBodyDecompressor : IMessageBodyDecompressor
+public class GzipMessageBodyCompression : IMessageBodyCompression
```

Comment:
We should add `///` docs to all the new public members in this PR before merging.